### PR TITLE
Update attack-surface-reduction-rules-deployment-test.md

### LINKED ISSUE #12612: Configure ASR rules per-rule exclusions
Configure ASR rules per-rule exclusions 

In the below section you can see we only adding that we can exclude the block action, while actually it's also exclude the Audit action from being triggered.

Open the [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431), and navigate to Home > Endpoint security > Attack surface reduction.
If it is not already configured, set the rule for which you want to configure exclusions to *Block.
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 498a23d9-d5b0-2c98-edfa-d9871675f5ae
* Version Independent ID: 498a23d9-d5b0-2c98-edfa-d9871675f5ae
* Content: [Test attack surface reduction (ASR) rules](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-deployment-test?view=o365-worldwide#configure-asr-rules-per-rule-exclusions)
* Content Source: [microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-deployment-test.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-deployment-test.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by msbemba**
Added audit or block to the sentence

fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12612

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-deployment-test.md [hunk 1]
```diff
@@ -196,7 +196,7 @@ ASR rules now provide the capability to configure rule-specific exclusions, know
 To configure per-rule exclusions:
 
 1. Open the [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431), and navigate to **Home** > **Endpoint security** > **Attack surface reduction**.
-1. If it is not already configured, set the rule for which you want to configure exclusions to **Block*.
+1. If it is not already configured, set the rule for which you want to configure exclusions to Audit or block.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-deployment-test.md]:
```suggestion
1. If it is not already configured, set the rule for which you want to configure exclusions to **Audit** or **Block**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**